javascript/builtins/Temporal/PlainDate.json

```diff
@@ -780,58 +780,6 @@
               }
             }
           },
-          "getISOFields": {
```

Comment:
Removed in https://github.com/tc39/proposal-temporal/commit/7899d5933cb9e15e187ea5ef2d1876c0c5cef73c (same for other `getISOFields` methods).